publicationiq.json

```diff
@@ -696,6 +703,158 @@
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          },
+          "502": {
+            "description": "Bad Gateway",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          },
+          "503": {
+            "description": "Service Unavailable",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "profile",
+            "in": "path",
+            "required": true,
+            "description": "Locale identifier. e.g. '1033'",
+            "type": "string"
+          },
+          {
+            "name": "localeid",
+            "in": "path",
+            "required": true,
+            "description": "EBSCO customer profile. e.g. 'demo.main.pfiguest'",
+            "type": "string"
+          },
+          {
+            "name": "sessionid",
+            "in": "query",
+            "description": "Already existing SessionId can be provided to improve experience of using the publications search. Example format -> '-2689046726050171609'",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "resourcetype",
+            "in": "query",
+            "required": false,
+            "description": "Resource type filter for alpha browser response. e.g. 'JOURNAL' only applies to title search type",
+            "type": "string",
+            "enum": [
+              "book",
+              "journal"
+            ]
+          },
+          {
+            "name": "searchType",
+            "in": "query",
+            "required": false,
+            "description": "Search type filter for alpha browser response. e.g. 'PACKAGE'",
+            "type": "string",
+            "enum": [
+              "TITLE",
+              "PACKAGE"
+            ]
+          },
+          {
+            "name": "menuid",
+            "in": "query",
+            "description": "Menu id. Used for filtering menus in position 1. Should be a positive number. e.g. '65'",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "password",
+            "in": "header",
+            "description": "authentication header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Authorization",

```

Comment:
I don't see a use for authorization header on the alphabrowse endpoint.

Reply:
@nwjohnson this endpoint accepts either a password or auth header just like the /publications endpoint

Reply:
Searching through the project and file for "Authorization", though, none of the other endpoints have that defined on it explicitly so unsure why we'd need it here.